### DISCUSSION TITLE
posts/raincloud-plot-alt/

### DISCUSSION THREAD

**Discussion by giscus** [mvuorre/mvuorre.github.io#24] [giscus]
# posts/raincloud-plot-alt/

I like raincloud plots, but think that they can duplicate the information a bit, which might have detrimental effects on clarity and comprehension.

https://vuorre.netlify.app/posts/raincloud-plot-alt/

**Comment by rmflight** [comment]
What about geom_sina from {ggforce}?? It bins the data, and then uses the data to fill in the density? I find them almost as good as a rain cloud plot, and follow the distributions better than the beeswarm.

Although I think the beeswarm / sina with the minimal distribution summary would be your best bet here, like take the summary from figure 3 and overlay it on the beeswarms in figure 4.

**Reply by mvuorre** [reply]
Thanks for the suggestion!

geom_sina() looks great, but I am specifically looking for one-sided alternatives. I think pairwise comparisons are visually easier if the "bottoms" of the distributions are next to one another.
